modules/task/execution_test.go

```diff
@@ -114,21 +117,33 @@ func TestEnqueueWithoutRegister(t *testing.T) {
 }
 
 func TestConsumeWithoutRegister(t *testing.T) {
+	host, _ := service.NewScopedHost(service.NopHost(), "task", "hello")
+	_testScope = host.Metrics()
+	_globalBackend = NewInMemBackend(host)
+	_errorCh = _globalBackend.(*inMemBackend).ErrorCh()
```

Comment:
not sure what you're talking about at all. explain in atleast a few sentences :) This is testing execution, not the cherami backend.